index.js

```diff
@@ -130,19 +141,31 @@ const { kConstruct } = require('./lib/web/cache/symbols')
 // in an older version of Node, it doesn't have any use without fetch.
 module.exports.caches = new CacheStorage(kConstruct)
 
-const { deleteCookie, getCookies, getSetCookies, setCookie } = require('./lib/web/cookies')
+const {
```

Comment:
This is the result of running `npm run lint:fix` on this branch. You may have something misconfigured if it reports diffs.

Reply:
Or this branch is behind and main has applied this formatting.

Reply:
please undo them

Reply:
Done. https://github.com/nodejs/undici/pull/3423#issuecomment-2410888383